src/schema/resolvers.js

```diff
@@ -40,53 +161,293 @@ const resolvers = {
       })
     },
 
-    updateUserLocation: async (_, args) => {
-      let userData = {}
-
+    updateUserLocation: async (_, { input }) => {
       // Finding if user exists
       if (
         !(await prisma.user.findUnique({
-          where: { userId: args.input.userId },
+          where: { userId: input.userId },
         }))
       )
         throw new Error('That user does not exist')
```

Comment:
This is a case where throwing an error makes more sense compared to `findThingById`

Reply:
To clarify, I think we _should_ throw an error if the given input prevents the resolver from completing its function, like with mutations.

With `getThingById`, an invalid id doesn't prevent the resolver from completing